server/server.py

```diff
@@ -182,6 +191,58 @@ async def verify(
 
     return {"results": results}
 
+@app.post("/verify_with_minibatch")
+async def verify_with_minibatch(
+    body: VerifyWithMiniBatchRequestBody,
+    access: require_access_dep,
+):
+    """verify the proof code by splitting into minibatches and using thread parrallel on each minibatch. Assumes header is the same for all proofs."""
+    codes = body.codes
+    timeout = body.timeout
+    mode = body.mode
+    buckets = body.buckets
+    subbatch_size = body.minibatch_size
+    infotree_type = body.infotree_type
+    disable_cache = body.disable_cache
+
+    minibatches = split_into_minibatches(codes, 5)
```

Comment:
We have codes be an array with single element today. 
The reason why `codes` was added is to avoid having to many network requests due to firewall restrictions. 

Do the mini-batches help if `len(codes) = 1` in benchmarks?